core/src/group16/project/game/views/GameScreen.kt

```diff
@@ -90,15 +101,15 @@ class GameScreen(val gameController: StarBattle, val fbic: FirebaseInterface) :
     fun endGame() {
         //This player won
         fbic.updateCurrentGameState(GameState.GAME_OVER)
+        var score: Int

```

Comment:
Idea: it is a possibly to check for draw too.

Reply:
I dont know what you mean by draw?

Reply:
Rip, bad wording from me. Tie (uavgjort) should be a better word. :P

Reply:
ahh okey, sorry. I fix

Reply:
Looks very good now! :D